frontend/iframe/viewmodels/RootViewModel.ts

```diff
@@ -271,6 +318,34 @@ export class RootViewModel extends ViewModel<SegmentType, Options> {
         });
     }
 
+    private async _showUnknownRoom(roomId: string) {
+        const client = new Client(this.platform);
+        let chosenSession;
+
+        let sessionInfos = await this.platform.sessionInfoStorage.getAll();
```

Comment:
I'm not sure I understand this, are we using a random session?

Reply:
I see what you mean, we could already be in a known session based on URL. Forgot about that working in WP builds.